samples/boards/st/power_mgmt/blinky/boards/nucleo_f103rb.conf

```diff
@@ -0,0 +1 @@
+CONFIG_COUNTER=y
```

Comment:
To be put under `soc/st/stm32/stm32f1x/Kconfig.defconfig` so it is set automatically when required:
```
if PM

config COUNTER
	default y

endif
```